syntax = "proto3";

package relay;

service Relay {
  rpc PlaceOrder (OrderRequest) returns (OrderResponse) {}
  rpc CancelOrder (OrderRequest) returns (OrderResponse) {}
}


message OrderRequest {
  string name = 1;
}


message OrderResponse {
  string message = 1;
}